client/src/game/Game.js

```diff
@@ -393,8 +393,13 @@ class Game extends Phaser.Scene {
     checkKeyFilters() {
         // Don't move while the chat input is open.
         if (GUIState.chatInputStatus) return true;
```

Comment:
This is probably not needed but I'm not sure.

Reply:
What is not needed?

Seems to work fine without any of the changes to this method.